docs/src/pages/guides/msw.md

```diff
@@ -156,3 +156,30 @@ import { setupServer } from 'msw/node';
 const server = setupServer();
 server.use(...getPetsMock());
 ```
+
```

Comment:
We need documentation for the property in the reference too.

https://orval.dev/reference/configuration/output#mock